@AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_steeringCursorDraw.sqf

```diff
@@ -35,7 +35,7 @@ if (cameraView != "GUNNER") then {_Offset = 0.14;};
 
 private _tex = ["\fza_ah64_us\tex\HDU\ah64_rkt.paa", "\fza_ah64_us\tex\HDU\ah64_rkt_fxd"] select (_sight == 3);
 _steeringCursorControl ctrlSetText _tex;
-_screenPos = [_Offset + deg (_heli animationPhase "tads_tur")*-4, 9.2 + (_pylonelev/0.64)] call fza_ihadss_fnc_angleToScreen;
+_screenPos = [_Offset + deg _tadsAzimuth*-4, 9.2 + (_pylonelev/0.64)] call fza_ihadss_fnc_angleToScreen;
```

Comment:
You're currently using a variable not defined in this function. This currently works because you are being called by the IHADSS draw code which has that variable in it, but if called by anywhere but there this'll end badly. Can you fetch please?

Reply:
As an aside, ensuring you don't accidentally do things like this is what the new command [`privateAll`](https://community.bistudio.com/wiki/privateAll) is for